cmake/radium_setup_functions.cmake

```diff
@@ -53,6 +53,7 @@ define_property(TARGET
 #   configure_cmdline_Radium_app(
 #         NAME theTargetName # <- this must be an executable
 #         RESOURCES ResourceDir1 ResourceDir2 # <- accept a list of directories
+#         [PREFIX prefix] # <- add prefix to resrouces see installTargetResources doc
 # )
 function(configure_cmdline_Radium_app)
```

Comment:
before merging, i'm for cleaning the naming convention here:
_ names, without up case.